### PR TITLE
Bulk download -> all files : show only fra regions and remove Global enhancement

### LINKED ISSUE #3885: Bulk download -> all files : show only fra regions and remove Global
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by sorja**
Screenshot 2024-08-26 at 16 07 31

### CODE REVIEW

File: src/server/controller/cycleData/getBulkDownload/index.ts [hunk 1]
```diff
@@ -73,11 +74,25 @@ const handleContent = async (content: Array<Record<string, string>>) => {
   return _convertToCSV(res)
 }
 
+const _getCountries = async (assessment: Assessment, cycle: Cycle) => {
+  const regionGroups = await RegionRepository.getRegionGroups({ assessment, cycle })
+  const fraRegions = Object.values(regionGroups).find((rg) => rg.name === 'fra2020')
```
Comment (minotogna) [inline src/server/controller/cycleData/getBulkDownload/index.ts]: Better to have an enum with region group names ?

File: src/server/controller/cycleData/getBulkDownload/index.ts [hunk 2]
```diff
@@ -73,11 +74,25 @@ const handleContent = async (content: Array<Record<string, string>>) => {
...
+  const allowedRegions = fraRegions?.regions.map((r) => r.regionCode)
+  const allCountries = await CountryRepository.getMany({ assessment, cycle })
+
+  return allCountries.reduce((acc, country) => {
+    if (!country.countryIso.startsWith('X')) {
```
Comment (minotogna) [inline src/server/controller/cycleData/getBulkDownload/index.ts]: I forgot , do we have Countries.isAtlantis ?

File: src/server/controller/cycleData/getBulkDownload/index.ts (hunk 1 above)
Comment (sorja) [reply src/server/controller/cycleData/getBulkDownload/index.ts]: Yes, was also thinking, will add this

meta/regionGroups/regionGroups.ts

File: src/server/controller/cycleData/getBulkDownload/index.ts (hunk 2 above)
Comment (sorja) [reply src/server/controller/cycleData/getBulkDownload/index.ts]: I will double check, if not, we should introduce it?
Should we always expect it to be `starts with X`?

Or should we also check for numbers? `/^X\d\d$`?

File: src/server/controller/cycleData/getBulkDownload/index.ts (hunk 1 above)
Comment (minotogna) [reply src/server/controller/cycleData/getBulkDownload/index.ts]: Don't we have an area folder ?

File: src/server/controller/cycleData/getBulkDownload/index.ts (hunk 2 above)
Comment (sorja) [reply src/server/controller/cycleData/getBulkDownload/index.ts]: https://github.com/openforis/fra-platform/pull/3891/commits/f58e0d1a2c4015a262429d1a1cdbac0e7e50d53b

File: src/server/controller/cycleData/getBulkDownload/index.ts (hunk 1 above)
Comment (sorja) [reply src/server/controller/cycleData/getBulkDownload/index.ts]: https://github.com/openforis/fra-platform/pull/3891/commits/f58e0d1a2c4015a262429d1a1cdbac0e7e50d53b

File: src/server/controller/cycleData/getBulkDownload/index.ts (hunk 2 above)
Comment (minotogna) [reply src/server/controller/cycleData/getBulkDownload/index.ts]: Starts with x is fine

File: src/server/controller/cycleData/getBulkDownload/index.ts (hunk 1 above)
Comment (sorja) [reply src/server/controller/cycleData/getBulkDownload/index.ts]: > Don't we have an area folder ?

yes, added there ! :)

**Review by minotogna — APPROVED** [review]

**Pull request merged**